deploy/src/main/deploy/kubernetes/common.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2018 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+
+set -e
+
+function is_olm_deployed {
+	kubectl get deployment olm-operator -n olm &> /dev/null && echo "YES" || echo "NO"
+}
+
+RESOURCES_BASE=https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/4370fc33c4ff7b6ba3a23def82d4db837dfcbb43/deploy/upstream/manifests/0.8.0/
```

Comment:
is there a particular reason why you want to keep this and the resource groups in this commons script? As far as I can see they are only used by the `olm_deploy.sh` script, right?

Does the `4370fc33c4ff7b6ba3a23def82d4db837dfcbb43` represent a particular commit in the repo, so that it is even more specific than just *master*?

Reply:
I initially had the plan to use those resources for deletion as well, however that doesn't seem to work, so I guess I can move them back to the `deploy` script.

It looks to me like as if the `0.8.0` directory itself gets updated over time, so I would really like to stick to a specific commit. This was the latest commit I saw for this directory.

Reply:
I moved the resources back to the `deploy` script.